recipes/pygypsy/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pygypsy" %}
+{% set version = "0.2.5" %}
+{% set md5 = "7d6d57e422e049974ac4d24da245655e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  skip: true  # [not py27]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - pygypsy = pygypsy.scripts.cli:cli
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - numpy x.x
+    - cython 0.25.1
+  run:
+    - python
+    - click >=6.6
+    - numpy >=1.11.2
```

Comment:
Though if it does only use NumPy greater than 1.11.2, maybe this requirement should be included in run as well to avoid attempting to build against 1.10.x unnecessarily.